app/views/application/personal-details/index.njk

```diff
@@ -76,28 +76,30 @@
     }]
   }) }}
 
-  {{ appAutocomplete({
-    label: {
-      text: "Nationality",
-      classes: "govuk-label--m"
-    },
-    items: nationalities
-  } | decorateApplicationAttributes(["candidate", "nationality"])) }}
-
-  {% set anotherNationality %}
+  {% if data.flags.nationality %}
```

Comment:
We will still want to ask for nationality in the MVP, but not do any branching branch based on this answer.